tests/post/steps/test_example_solution.py

```diff
@@ -0,0 +1,331 @@
+    # 1. Solution Configuration is created at /etc/metalk8s/solution.yaml
+    # 2. Solution Configuration has new archive path
+    # 3. Solutions ConfigMap has new Solution imported
+
+    with host.sudo():
+        assert host.file(SOLUTION_CONFIGURATION_FILE).exists, \
+            'No Solution Configuration file exists at {}'.format(
+                SOLUTION_CONFIGURATION_FILE
+            )
+    # Todo: SOLUTION_CONFIGURATION_FILE is yaml, we could parse it and ensure
+    # the archive path is at the right location
+    with host.sudo():
+        assert name in host.file(SOLUTION_CONFIGURATION_FILE).content_string, \
+            'No {} Solution ISO path found in Solution Config {}'.format(
+                name, SOLUTION_CONFIGURATION_FILE
+            )
+    configmap = get_configmap(
+        k8s_client, SOLUTION_CONFIGMAP, SOLUTION_NAMESPACE
+    )
+    assert name in configmap.get('data', {}), \
+        '{} Configmap does not contain {}'.format(SOLUTION_CONFIGMAP, name)
+
+
+@then(parsers.parse("Solution '{name}' is available"))
+def is_available_solution(host, request, name):
+    assert name in get_available_solution(host)
+
+
+@then(parsers.parse("solution environment '{name}' is available"))
+def read_solution_environment(k8s_client, name):
+    solution_environment_obj = get_environment(k8s_client, name)
+    assert solution_environment_obj is not None
+
+
+@then(parsers.parse(
+    "we have '{value}' available '{path}' for Solution '{name}' Deployment in "
+    "environment '{environment}'"))
+def read_solution_deployments(
+    k8s_appsv1_client, value, path, name, environment
+):
+    def _wait_for_deployment():
+        response = get_deployment(k8s_appsv1_client, name, environment)
+        available_replicas = utils.get_dict_element(response, path)
+        assert available_replicas is not None
+
+        assert literal_eval(value) == available_replicas, (
+            "Expected {} replicas for Deployment {}, got {}".format(
+                value, name, context.get('deployment_obj')
+            )
+        )
+    utils.retry(
+        _wait_for_deployment,
+        times=10,
+        wait=5,
+        name="wait for available Deployments"
+    )
+
+
+@then(parsers.parse("we have no Solution '{name}' archive mounted"))
+def no_solution_mountpoint(host, name, k8s_client):
+    with host.sudo():
+        assert name not in host.mount_point.get_mountpoints(), \
+            'Found Solution mountpoint {}'.format(name)
+
+
+@then(parsers.parse("we have no Solution environment '{name}'"))
+def no_solution_environment(k8s_client, name):
+    solution_environment_obj = get_environment(k8s_client, name)
+    assert solution_environment_obj is None
+
+
+@then(parsers.parse("we have no available Solution '{name}'"))
+def no_available_solution(host, name):
+    assert name not in get_available_solution(host)
+
+
+@then(parsers.parse("we have no Solution Configuration file present"))
+def no_solution_config(host, name):
```

Comment:
No need for parsers.parse() if there is nothing to parse.
And from where does the `name` parameter comes from ?